src/corona_hakab_model/main.py

```diff
@@ -30,7 +30,8 @@ def main():
 
     parser.add_argument(
         "-MSM", "--medical-state-machine", dest="medical_state_machine_params",
-        help="Parameters for the medical state machine"
+        help="Parameters to build the medical state machine",
+        default=str(Path(MODEL_FOLDER) / "default_configuration" / "medical_state_machine.json")
```

Comment:
why not just do os.path.join instead of this convolution?

Reply:
The goal was to keep it independent of the location of this file’s location.